### PR TITLE
Release announcement/update for 3.1

### PR DESCRIPTION
**Pull request by eteq**
Just what it says on the tin... To be merged when the 3.1 builds are all done.

### CODE REVIEW

File: announcements/release-3.1.html [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+		Astropy is a community-driven Python package intended to contain much of the
+		core functionality and common tools needed for astronomy and astrophysics.
+		It is part of the Astropy Project, which aims to foster an ecosystem of
+		interoperable astronomy packages for Python.
+		</p>
+		<p>
+		Thus release had a performance focus, but also has new and improved major functionality.  Highlights include:
+		<ul>
+			<li>Performance Improvements across-the-board</li>
+			<li>A new Sub-package for Uncertainties and Distributions</li>
```
Comment (bsipocz) [inline announcements/release-3.1.html]:
```suggestion
			<li>A new sub-package for Uncertainties and Distributions</li>
```

File: announcements/release-3.1.html [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+		</ul>
+		</p>
+		<p>
+		In addition, hundreds of smaller improvements and fixes have been made. An overview of the changes is provided at:
+		</p>
+		<p>
+		&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://docs.astropy.org/en/stable/whatsnew/3.1.html">http://docs.astropy.org/en/stable/whatsnew/3.1.html</a>
+		</p>
+		<p>
+		Note that the Astropy 3.x series only supports Python 3.  Python 2 users can continue to use the 2.x series, which will receive bug fixes and support until the Python developers permanently sunset Python 2.7  (scheduled for 2019).
```
Comment (bsipocz) [inline announcements/release-3.1.html]:
```suggestion
		Note that the Astropy 3.x series only supports Python 3.  Python 2 users can continue to use the 2.x series, which will receive bug fixes until the Python developers permanently sunset Python 2.7  (scheduled for 2019).
```

File: announcements/release-3.1.html [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+		&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://github.com/astropy/astropy/issues">https://github.com/astropy/astropy/issues</a>
+		</p>
+		<p>
+		Over 253 developers have contributed code to Astropy so far, and you can find out more about the team behind Astropy here:
+		</p>
+		<p>
+		&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://www.astropy.org/team.html">http://www.astropy.org/team.html</a>
+		</p>
+		<p>
+			As a reminder, Astropy v2.0 (our long term support release) will continue to be supported with bug fixes until the end 2019, so if you need to use Astropy in a very stable environment, you may want to consider staying on the v2.0.x set of releases (for which we have recently released v2.0.4).
```
Comment (bsipocz) [inline announcements/release-3.1.html]:
```suggestion
			
```

File: announcements/release-3.1.html [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+		<p>
+		&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://github.com/astropy/astropy/issues">https://github.com/astropy/astropy/issues</a>
+		</p>
+		<p>
+		Over 253 developers have contributed code to Astropy so far, and you can find out more about the team behind Astropy here:
+		</p>
+		<p>
+		&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://www.astropy.org/team.html">http://www.astropy.org/team.html</a>
+		</p>
+		<p>
```
Comment (bsipocz) [inline announcements/release-3.1.html]:
```suggestion
		
```

File: announcements/release-3.1.html [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+			As a reminder, Astropy v2.0 (our long term support release) will continue to be supported with bug fixes until the end 2019, so if you need to use Astropy in a very stable environment, you may want to consider staying on the v2.0.x set of releases (for which we have recently released v2.0.4).
+
+		</p>
```
Comment (bsipocz) [inline announcements/release-3.1.html]:
```suggestion
		
```

File: announcements/release-3.1.html (hunk 2 above)
Comment (bsipocz) [inline announcements/release-3.1.html]: Maybe also emphasize that the 2.x series are not receiving new features?

Comment (eteq) [reply announcements/release-3.1.html]: Oops, I got this backward.  Why exactly did you want to delete this?  I think we *have* to promise support here.

File: announcements/release-3.1.html [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+				<li><a href="../help.html">Get Help</a></li>
+				<li><a href="../contribute.html">Contribute</a></li>
+				<li>
+					<div class="dropdown">
+						<a href="http://docs.astropy.org">Documentation</a>
+						<div class="dropdown-content">
+							<ul>
+								<li><a href="http://docs.astropy.org" target="_blank">Current Release</a></li>
+								<li><a href="http://devdocs.astropy.org/" target="_blank">In Development</a></li>
+								<li><a href="https://astropy.readthedocs.io/en/v1.2.2/index.html" target="_blank">v1.2.2</a></li>
```
Comment (bsipocz) [inline announcements/release-3.1.html]: @eteq - it still has the full dropdown, please change it to the text only 3 options that was discussed in Adrian's PR.

File: announcements/release-3.1.html (hunk 2 above)
Comment (eteq) [reply announcements/release-3.1.html]: I would still consider that support... but take your point - will push something up in a bit with other wording.

**Comment by eteq** [comment]
@adrn - maybe have a quick look at the updates I made to the dropdown and make sure you're 👍 ? (key point: turns out the dropdowns were mis-matched in a *lot* of places, but now they are better, since we don't have to worry about updating version numbers any more.)

File: announcements/release-3.1.html [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
+		    <a href="http://www.astropy.org">http://www.astropy.org</a>
+		</p>
+		<p>
+		Astropy is a community-driven Python package intended to contain much of the
+		core functionality and common tools needed for astronomy and astrophysics.
+		It is part of the Astropy Project, which aims to foster an ecosystem of
+		interoperable astronomy packages for Python.
+		</p>
+		<p>
+		Thus release had a performance focus, but also has new and improved major functionality.  Highlights include:
```
Comment (adrn) [inline announcements/release-3.1.html]:
```suggestion
		The focus of this release is on performance, but it also contains new and improved major functionality.  Highlights include:
```

File: announcements/release-3.1.html [hunk 8]
```diff
@@ -0,0 +1,207 @@
...
+		<ul>
+			<li>Performance Improvements across-the-board</li>
```
Comment (adrn) [inline announcements/release-3.1.html]:
```suggestion
			<li>Performance improvements across-the-board</li>
```

File: announcements/release-3.1.html [hunk 9]
```diff
@@ -0,0 +1,207 @@
...
+			<li>A new sub-package for Uncertainties and Distributions</li>
+			<li>A new Box Least Squares Periodogram</li>
+			<li>J-style coordinates parser</li>
+			<li>Support for use of Time and TimeDelta columns within a Table</li>
+			<li>Array-valued Time and TimeDelta objects are now mutable</li>
+			<li>Better uncertainty support and bit planes in NDData</li>
+			<li>A new operator for quantity performance</li>
```
Comment (adrn) [inline announcements/release-3.1.html]:
```suggestion
			<li>A new operator for Quantity performance</li>
```

File: announcements/release-3.1.html [hunk 10]
```diff
@@ -0,0 +1,207 @@
...
+			<li>Thermodynamic temperature equivalency</li>
+			<li>Little-h equivalency</li>
+			<li>Change in default cosmology</li>
+			<li>Significant improvements in WCSAxes</li>
+			<li>New convenience function for imshow with ImageNormalize</li>
```
Comment (adrn) [inline announcements/release-3.1.html]:
```suggestion
			<li>A new convenience function for imshow with ImageNormalize</li>
```

File: announcements/release-3.1.html [hunk 11]
```diff
@@ -0,0 +1,207 @@
...
+		</p>
+		<p>
+		&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://www.astropy.org/team.html">http://www.astropy.org/team.html</a>
+		</p>
+		<p>
+			As a reminder, Astropy v2.0 (our long term support release) will continue to be supported with bug fixes (but no new features) until the end of 2019, so if you need to use Astropy in a very stable environment, you may want to consider staying on the v2.0.x set of releases (for which we have recently released v2.0.10).
+
+		</p>
+		<p>
+		If you use Astropy directly for your work, or as a dependency to another
```
Comment (adrn) [inline announcements/release-3.1.html]: Why not copy and paste the acknowledgement from the [acknowledging and citing](http://www.astropy.org/acknowledging.html) page, or link to that?

**Comment by eteq** [comment]
> Drop-down looks ok, though eventually we should find a solution for the menu bar that doesn't involve copy-pasta!

I agree in spirit, but it's hard to combine that with static web pages, unfortunately... I think this is why PHP was invented :wink:.  But maybe we can try at some point transitioning to hyde or some similar static site generator? If that's a direction to go, we should probably make an issue...

At any rate, I think I've updated this based on your comments, @adrn.

**Comment by eteq** [comment]
(oops, I screwed things up with a rebase/force push, one moment please)

**Pull request merged**